### DISCUSSION TITLE
'Send to openoutpaint' button doesn't seem to be working.

### DISCUSSION THREAD

**Discussion by webbedgiant** [zero01101/openOutpaint#218] [Q&A]
I try sending a txt2img generation over to openoutpaint and nothing happens after selecting the button. Could anyone help?

**Comment by zero01101** [comment]
yeah, the gradio update to webUI caused a bit of havoc with extensions - should be fixed as of https://github.com/zero01101/openOutpaint-webUI-extension/commit/566a8a53c910f902297c4926f0d3634900403e53
